test/functions/extension.test.ts

```diff
@@ -0,0 +1,57 @@
+import {bool, dateTime, merge, numeric, str, StringMap, stringToTermPrefix, TermMap, wrap} from '../util/Aliases';
+import {Notation, testTable} from '../util/TruthTable';
+import * as RDF from 'rdf-js';
+import {NamedNode} from 'rdf-js';
+
+function mapToTerm(map: StringMap): TermMap {
+  return Object
+    .keys(map)
+    .reduce((p, k) => ({ ...p, [k]: stringToTermPrefix(map[k]) }), {});
+}
+
+describe('extension function: term-equal', () => {
```

Comment:
Can we also have a test case that checks a custom uppercase function?

Reply:
This would be pretty hard to implement as of right now but we should probably test this when we look at the tests.